Jenkinsfile

```diff
@@ -164,7 +164,7 @@ pipeline {
                                 sh """
                                     set -e
                                     rm -rf src/brave
-                                    npm run init -- --target_os=android
+                                    npm run init -- --target_os=android --target_arch=arm64
```

Comment:
target_arch is not a valid parameter for `init`. Only target_os is required here

Reply:
thought it was from https://github.com/brave/brave-browser/blob/master/scripts/sync.js#L16, removing now